google/cloud/bigquery/v2/minimal/internal/job_request.cc

```diff
@@ -70,6 +71,25 @@ StateFilter StateFilter::Done() {
   return state_filter;
 }
 
+std::string GetJobRequest::DebugString(TracingOptions const& options) const {
+  std::string out;
+  absl::StrAppend(&out, "GetJobRequest{project_id=", project_id_,
+                  ", job_id=", job_id_, ", location=", location_, "}");
+  return internal::DebugString(out, options);
```

Comment:
We want to actually use the TracingOptions. There is one about breaking a message onto multiple lines and another about truncating string fields.

This is an example that I know of. It seems to reveal that our tracing options were "inspired" by protobuf's print settings: https://github.com/googleapis/google-cloud-cpp/blob/952a50ddadb2dfc62536ccc02afb690afaa67ee6/google/cloud/internal/debug_string_protobuf.cc#L67-L85

We want to truncate **individual string fields** in the message and split the fields of the message by newline vs. space.

I think we want something like:
```cc
auto delim = options.single_line_mode() ? " " : "\n";
return absl::StrAppend(&out, "GetJobRequest{", delim, "project_id=", DebugString(project_id_, options), delim, "job_id=", DebugString(job_id_, options), delim, ...., delim, "}");
```

(I did not try to compile this thing.)